File: Plugins/Aspose_Words_for_PHP/src/quickstart/loadandsavetodisk/php/LoadAndSaveToDisk.php
<?php
/*
 * Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Words. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */

class LoadAndSaveToDisk {

    public  static function saveToDisk() {

        // The path to the documents directory.
        $dataDir = "/usr/local/apache-tomcat-8.0.22/webapps/JavaBridge/Aspose_Words_Java_For_PHP/src/quickstart/loadandsavetodisk/data/";
        // Load the document from the absolute path on disk.
        $doc = new Java("com.aspose.words.Document", $dataDir . "Document.doc");
        // Save the document as DOCX document.");
        $doc->save($dataDir . "Document Out.docx");

    }

}

File: Plugins/Aspose_Words_Java_for_Ruby/lib/asposewordsjavaforruby/loadandsavetodisk.rb
module Asposewordsjavaforruby
  module LoadAndSaveToDisk
    def initialize()
        # Load and save the document.
        save_to_disk()
    end
        
    def save_to_disk()
        # The path to the documents directory.
        data_dir = File.dirname(File.dirname(File.dirname(__FILE__))) + '/data/quickstart/'

        # Open the document.
        doc = Rjb::import('com.aspose.words.Document').new(data_dir + "Document.doc")

        # Save the document as DOCX document.
        doc.save(data_dir + "Document Out.docx")
    end 

  end
end
